src/array_api_stubs/_draft/fft.py

```diff
@@ -387,7 +387,7 @@ def rfftn(
     Returns
     -------
     out: array
-        an array transformed along the axes (dimension) indicated by ``axes``. The returned array must have a complex-valued floating-point data type determined by :ref:`type-promotion`.
+        an array transformed along the axes (dimension) indicated by ``axes``. The returned array must have a complex floating-point data type whose precision matches the precision of ``x`` (e.g., if ``x`` is ``float64``, then the returned array must have a ``complex128`` data type). The length along each transformed axis ``i`` is ``s[i]``, if ``s`` is given and ``s[i]`` is not ``-1``, or the length of the corresponding axis of the input array otherwise.
```

Comment:
This doesn't match what NumPy does. From [the NumPy docs for `rfftn`](https://numpy.org/doc/stable/reference/generated/numpy.fft.rfftn.html):

> The length of the last axis transformed will be s[-1]//2+1, while the remaining transformed axes will have lengths according to s, or unchanged from the input.

(here `s` defaults to the shape of the input along the given axes if not given)

Reply:
Yes, would you mind fixing it Aaron?